packages/transformers/html/src/dependencies.js

```diff
@@ -129,7 +129,8 @@ export default function collectDependencies(
 
     if (tag === 'link' && attrs.rel === 'manifest' && attrs.href) {
       attrs.href = asset.addURLDependency(attrs.href, {
-        isEntry: true
+        isEntry: true,
+        env: {context: 'pwa-manifest'}
```

Comment:
I did rebase and now get:
```
Error: Received URL without a pathname.
```

something with `moduleSpecifier` (new api?)

How do I properly add `manifest` dep here?